packages/lesswrong/server/search/elastic/ElasticSearchController.ts

```diff
@@ -0,0 +1,39 @@
+import { Application, Request, Response, json } from "express";
+import { isValidSearchQuery } from "./searchQuery";
+import ElasticSearchService from "./ElasticSearchService";
+
+class ElasticSearchController {
```

Comment:
Does this need to be a class?  We aren't using the constructed instance, it seems to purely be for assigning a route handler.  Could be handled the same we handle other non-graphql API routes?